agent/src/main/resources/reference.conf

```diff
@@ -67,6 +67,6 @@ kanela {
   }
   # Allow replace classes
   class-replacer {
-    replace = []
+    replace = [] ${?kamon.class-replacer.replace}
```

Comment:
Why?

Reply:
This is a weird case on the conf loading order, in this line I try to concat to any existent list on this key an empty list.
Instead of just define the property as an empty list, depending on the order of loading if I leave just the `[]` definition, this could override any property already setted by another jar.
I need to leave the `[]` instead of just delete the config key because if no one set the key we will have a default value to work with.